MODULE.bazel

```diff
@@ -11,6 +12,46 @@ go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
 
 go_sdk.download(version = "1.24.1")
 
+git_override(
+    module_name = "gazelle_kt",
+    remote = "https://github.com/alexeagle/gazelle_kt.git",
+    commit = "3d883c03f9a69b9d0d013ba9f2acc7cb049ef50e",
+)
+
+go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
+
+go_deps.module_override(
+    patch_strip = 1,
+    patches = [
+        "patches/aspect-cli.only-kotlin.patch",
```

Comment:
Is there an issue we can point to on `gazelle_kt` that will allow dropping these?

Reply:
I didn't bother filing one, we can either fix that before this lands or after